Stacks the papers inside a folder inside briefcase

SouDescolado opened the pull request:
Briefcases now spawn with a pen and a blue folder, filled with papers, instead of spawning filled with papers.

SouDescolado committed:
Stacks the papers inside a folder inside briefcase

Makes all paper be neatly stacked inside a folder in the briefcase.

SouDescolado committed:
Stacks the papers inside a folder inside briefcase

Briefcases now spawn with a pen and a blue folder, filled with papers, instead of spawning filled with papers.

SouDescolado committed:
Update briefcase.dm

KorPhaeron commented:
Neat

Bawhoppen commented:
Make it spawn with a law office stamp too

SouDescolado commented:
Will do. Let me fix it first

SouDescolado committed:
Update briefcase.dm

SouDescolado committed:
Update briefcase.dm

Fixed travis(?), added law stamp to briefcase

SouDescolado committed:
Update briefcase.dm

gah

SouDescolado committed:
Update briefcase.dm

Removed Law Stamp from briefcases (Not all briefcases spawn at the law office)
Tried to bribe travis with cookies.

PKPenguin321 commented:
seventh time's the charm i guess

SouDescolado commented:
It was the cookies.

Bawhoppen commented:
you forgot the stamp :frowning:

SouDescolado commented:
I added, then removed when someone mentioned on IRC to remember that not all briefcases spawn at the law office. Maybe make it spawn at the Lawyer's right pocket?

Bawhoppen commented:
make it a different briefcase than cause blue folder = law office

SouDescolado commented:
Shit...

SouDescolado committed:
Changing folder to generic yellow color

Cookies to Travis.

Bawhoppen commented:
yellow is cargo though I believe

SouDescolado commented:
There are no other avaliable colors, are there? Should I just remove the /yellow/ then?

Jalleo commented:
There are red folders too check the bearocacy.dmi icon file i think. (cant spell it but you should get what i mean.)

SouDescolado commented:
Arent red security? (And white medical)

Bawhoppen commented:
I'm saying you should just make a seperate briefcase for just lawyers

Jalleo commented:
Technically law office is somewhat sec somewhat civilian could give prosecutor preset outfit laywer red defence blue.
It would be a slight reference to basically anything and everyone would probably be fine with it.

Jalleo commented:
Also what @Bawhoppen says if we get a brown folder sprite for standard briefcases.

Bawhoppen commented:
I can get a recolored sprite but what the fuck are you talkin about Jalleo

Bawhoppen commented:
the grey ones are the basic ones I just remembered

SouDescolado commented:
I think if I just remove the color tag (leaving folder/folder/ instead of folder/blue/folder) will get the generic folder.

Jalleo commented:
All sorted then. Also never seen grey ones..

Bawhoppen commented:
thats cause on station most are department colored

SouDescolado commented:
I'm fixing guys. Getting (even more) help from the pros

Jalleo commented:
If i was on irc right now i would be helping blame the bus.

SouDescolado committed:
Fixes folder color. Defined a new briefcase type

Cookies for travis
Need to fix lawyers spawned briefcase. Will try to fix it.

SouDescolado committed:
Changes lawyer briefcase

Gives a special spaceman a special briefcase (Actually a normal briefcase with other contents)

SouDescolado committed:
Update briefcase.dm

Complete code overhaul

SouDescolado committed:
Update briefcase.dm

Fixed bogus
Cookies to travis

PKPenguin321 commented:
>12 commits

At least you're doing better than my second-ever PR

SouDescolado commented:
Well, when you are learning, you have to code, fix the code, make the code better, add new things to the code, curse because the code doesnt work, throw away the code and make new code. I'm lucky it only took 12 commits.

tkdrg commented on code/game/objects/items/weapons/storage/briefcase.dm:
```diff
@@ -13,17 +13,22 @@
+
+/obj/item/weapon/storage/briefcase/lawyer/New()
+	new /obj/item/weapon/stamp/law(src)
+	..()
```
Shouldn't these two be the other way around?
```
	..()
	new /obj/item/weapon/stamp/law(src)
```

SouDescolado replied:
This way, it will first get the law stamp, then the pen and the folder. The other way it would get the pen, the folder, then the law stamp. Both ways would work, if I'm not mistaken.

tkdrg replied:
Yeah I guess, the latter just made more sense in my head. But sure.
